File: backend/src/main/java/com/ftn/activityapp/dto/UserResponse.java
package com.ftn.activityapp.dto;

import com.ftn.activityapp.enums.ActivityLevel;
import com.ftn.activityapp.enums.UserGoal;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class UserResponse {
    private Long id;
    private String email;
    private String firstName;
    private String lastName;
    private Integer age;
    private Double height;
    private Double weight;
    private ActivityLevel activityLevel;
    private UserGoal goal;
}

File: mobile-app/app/src/main/java/com/example/activityapp/data/remote/dto/UserResponse.kt
package com.example.activityapp.data.remote.dto

data class UserResponse(
    val id: Long,
    val email: String,
    val firstName: String,
    val lastName: String,
    val age: Int,
    val height: Double,
    val weight: Double,
    val activityLevel: String,
    val goal: String,
    val goalSteps: Int
)